package.json

```diff
@@ -102,51 +96,74 @@
     "cpy-cli": "^3.1.1",
     "css-color-converter": "^2.0.0",
     "cssnano": "^4.1.10",
+    "esbuild": "^0.12.9",
+    "esbuild-plugin-svgr": "^0.0.2",
     "eslint-config-airbnb-typescript": "^12.0.0",
     "eslint-config-react-app": "^6.0.0",
     "eslint-plugin-mdx": "^1.8.2",
     "eslint-plugin-prettier": "^3.3.1",
+    "fs-extra": "^10.0.0",
+    "glob": "^7.1.7",
     "husky": "^4.3.8",
     "jest-css-modules": "^2.1.0",
-    "postcss": "^7",
+    "path": "^0.12.7",
+    "postcss": "^8.3.5",
     "postcss-cli": "^7.1.1",
-    "postcss-comment": "^2.0.0",
-    "postcss-each": "^0.10.0",
-    "postcss-functions": "3.0.0",
-    "postcss-import": "12.0.0",
-    "postcss-nested": "^4.2.3",
-    "postcss-scss": "^2.1.1",
-    "postcss-simple-vars": "5.0.2",
+    "postcss-focus-visible": "^5.0.0",
+    "postcss-functions": "^4.0.2",
+    "postcss-import": "^14.0.2",
+    "postcss-loader": "^6.1.0",
+    "postcss-scss": "^4.0.0",
     "postcss-strip-inline-comments": "^0.1.5",
-    "postcss-url": "8.0.0",
+    "postcss-url": "^10.1.3",
     "precss": "^4.0.0",
     "prettier": "^2.2.1",
     "react-is": "^17.0.1",
     "rollup-plugin-postcss": "^3.1.8",
     "rollup-plugin-url": "^3.0.1",
-    "sass-loader": "^10.1.1",
     "size-limit": "^4.9.2",
     "string-dedent": "^2.0.0",
-    "tailwindcss": "npm:@tailwindcss/postcss7-compat",
-    "tsdx": "^0.14.1",
+    "tmp": "^0.2.1",
     "tslib": "^2.1.0",
-    "typescript": "^4.1.3"
+    "typescript": "^4.1.3",
+    "util": "^0.12.4"
   },
   "files": [
     "dist",
     "src"
   ],
   "engines": {
-    "node": ">=10"
+    "node": ">=14"
   },
   "peerDependencies": {
-    "formik": ">=2.2.6",
-    "react": ">=16"
+    "@date-io/luxon": "^2.10.8",
+    "@material-ui/core": "^4.11.3",
+    "@material-ui/lab": "^4.0.0-alpha.57",
+    "@material-ui/pickers": "4.0.0-alpha.12",
+    "@stripe/react-stripe-js": "^1.2.2",
+    "@stripe/stripe-js": "^1.11.0",
+    "@szhsin/react-menu": "^1.9.0",
+    "chart.js": "^2.9.4",
+    "chartjs-plugin-datalabels": "^0.7.0",
+    "classnames": "^2.2.6",
+    "clipboard": "^2.0.6",
+    "formik": "2.2.6",
+    "framer-motion": "^3.2.1",
+    "lodash": "^4.17.20",
+    "luxon": "^1.25.0",
+    "notistack": "^1.0.3",
+    "postcss-focus-visible": "^5.0.0",
+    "react": "16.13.1",
+    "react-chartjs-2": "^2.11.1",
+    "react-dom": "16.13.1",
+    "react-input-mask": "^2.0.4",
+    "react-modal": "^3.12.1",
+    "react-quill": "2.0.0-beta.2",
+    "react-text-mask": "^5.4.3",
+    "yup": "^0.32.8"
   },
   "husky": {
-    "hooks": {
-      "pre-commit": "tsdx lint src"
-    }
+    "hooks": {}
   },
```

Comment:
Should we just remove the `husky` setting altogether?

Reply:
idk, we might want to utilize for something later?